packages/ui/src/filter/filter-list.tsx

```
@@ -337,5 +356,283 @@ function FilterCombobox({
+          )}
+        </button>
+      </Popover>
+
+      <button
+        type="button"
+        className="h-full rounded-r-md p-2 text-neutral-500 ring-inset ring-neutral-500 hover:bg-neutral-100 hover:text-neutral-800 focus:outline-none focus-visible:ring-1"
+        onClick={() => {
+          if (onRemoveFilter) {
+            onRemoveFilter(filterKey);
+          } else {
+            values.forEach(value => onRemove(filterKey, value));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)